packages/compass-components/jest.config.js

```diff
@@ -0,0 +1,14 @@
+module.exports = {
```

Comment:
I'd put this in the ROOT too, if that's the direction i think it doesn't hurt to just start that way